client/state/sites/reducer.js

```diff
@@ -125,24 +127,60 @@ export function items( state = {}, action ) {
 			};
 		}
 
-		case SITE_ICON_SET: {
-			const { siteId, iconUrl } = action;
+		case SITE_SETTINGS_UPDATE:
+		case SITE_SETTINGS_RECEIVE: {
```

Comment:
So, If I understand properly we're receiving the site Icon ID from the site settings endpoint, I'm wondering why we're storing it in this reducer, instead of relying on the siteSettings state to handle it (like other settings)?